evals/scripts/bench_significance.py

```
@@ -0,0 +1,399 @@
+        f"Report: {path.name}",
+        f"  model={cfg.get('model')}  judge={cfg.get('judge_model')}  "
+        f"dataset={cfg.get('dataset')}  seed={cfg.get('seed')}  rules={stats['rules']}",
+        "",
+        f"Verdicts (N={stats['n']}):  mcp={stats['mcp']}  vanilla={stats['vanilla']}  ties={stats['ties']}",
+        f"Decisive (ties excluded): {stats['decisive']}  →  MCP share = {stats['mcp_decisive_share']:.1%}",
+        f"  two-sided exact binomial p (vs coin flip) = {stats['binom_p']:.4f}"
+        f"  [{'significant' if sig else 'NOT significant'} at α=0.05]",
+        f"  Wilson 95% CI on MCP share = [{lo:.1%}, {hi:.1%}]",
+        f"  DECISION GATE (lower bound > 50%): {'PASS — MCP wins' if ships else 'not met'}",
+        "",
+        f"Methodology health:",
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)